test/OpenTelemetry.Instrumentation.Dependencies.Tests/HttpWebRequestActivitySourceTests.net461.cs

```diff
@@ -177,17 +177,17 @@ public async Task TestBasicReceiveAndResponseEvents(string method, string queryS
             Assert.Equal(1, eventRecords.Records.Count(rec => rec.Key == "Stop"));
 
             // Check to make sure: The first record must be a request, the next record must be a response.
-            (Activity activity, HttpWebRequest startRequest) = AssertFirstEventWasStart(eventRecords);
+            var wrapper = AssertFirstEventWasStart(eventRecords);
```

Comment:
I think if you add...
`<PackageReference Include="System.ValueTuple" Version="4.5.0" Condition="'$(TargetFramework)' == 'net452'" />`
...to the csproj than the Tuple syntax would have worked.

Reply:
i will change that and undo. thanks for the advice